crmsh/sbd.py

```diff
@@ -551,18 +551,19 @@ def join_sbd(self, peer_host):
 
         if not utils.package_is_installed("sbd"):
             return
-        if not os.path.exists(SYSCONFIG_SBD) or not utils.service_is_enabled("sbd.service", peer_host):
```

Comment:
The only thing changed in this line is coding style. Is there any consensus on it?